hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -205,6 +205,18 @@
   // - 2 for OTBN test,
   // This allows an arbitrary number of SW images to be supplied to the TB.
   tests: [
+    {
+      name: chip_sw_example_flash_test
```

Comment:
nit: we are following the convention of not adding the `_test` suffix, since all of these are already known to be "tests".

Reply:
Ah right, my bad. Fixed.